src/main/java/seedu/duke/ui/Ui.java

```diff
@@ -22,9 +22,13 @@ public void showMainScreen() {
         System.out.print(Messages.MESSAGE_MAIN_PROMPT);
     }
 
-    public void showTaskScreen(String patientName) {
+    public void showTaskScreen(String patientName, String patientTag) {
+        String tagFormatted = "";
+        if (patientTag != null) {
+            tagFormatted = " [" + patientTag + "]";
+        }
         showLine();
-        System.out.println("Patient: " + patientName);
+        System.out.println("Patient: " + patientName + tagFormatted);
```

Comment:
How about passing in Patient object? This is allow to get patient `name` and `getFormattedTag`

Reply:
good idea. will change it to the suggested implementation